### PR TITLE
Update weave-net to v2.6.0

### PR DESCRIPTION
**Pull request by lnovara**
Nothing special, just updated weave-net to v2.5.2.

Please note that I have added the `IPALLOC_RANGE` environment variable with a value matching the default for `kubernetes_pod_cidr` in the `kube-control-plane` Ansible role of [fury-kubernetes-aws](https://github.com/sighupio/fury-kubernetes-aws/blob/master/roles/kube-control-plane/defaults/main.yml).

### CODE REVIEW

**Comment by angelbarrera92** [comment]
Hi @lnovara. What about update weave-net to: https://github.com/weaveworks/weave/releases/tag/v2.6.0 ?

Thanks

**Comment by lnovara** [comment]
Makes sense, I have only to check if we need further modify the manifests beside updating the image tag.

**Comment by lnovara** [comment]
@angelbarrera92 I've update weave-net to latest stable and in the meantime I've improved the `tolerations` and increased the priority of weave-net pods.

**Comment by angelbarrera92** [comment]
LGTM!

Minor things in the upcoming comments

File: katalog/weave-net/README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ extra code to run.
 
 ## Image repository and tag
 
-* Weave Net image: `weaveworks/weave-kube:2.4.1`
-* Weave Net NPC image: `weaveworks/weave-npc:2.4.1`
+* Weave Net image: `weaveworks/weave-kube:2.5.2`
```
Comment (angelbarrera92) [inline katalog/weave-net/README.md]: Change it to 2.6.0

File: katalog/weave-net/kustomization.yaml [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
-  - rbac.yml
-  - deploy.yml
-  - sm.yml
\ No newline at end of file
+- ds.yml
+- rbac.yml
+- rules.yml
+- sm.yml
+
+imageTags:
```
Comment (angelbarrera92) [inline katalog/weave-net/kustomization.yaml]: Cool!

File: katalog/weave-net/README.md (hunk 1 above)
Comment (lnovara) [reply katalog/weave-net/README.md]: Fixed

File: katalog/weave-net/kustomization.yaml (hunk 2 above)
Comment (lnovara) [reply katalog/weave-net/kustomization.yaml]: ☺️

**Comment by angelbarrera92** [comment]
LGTM!

**Comment by lnovara** [comment]
I will proceed to merge this PR.

@angelbarrera92 what do you think about the release? Minor or patch? Personally, I would go for a minor since we are upgrading an existing component.

**Comment by ralgozino** [comment]
There's also to update the weave-net version in the repo's main README.md file

**Comment by angelbarrera92** [comment]
> I will proceed to merge this PR.
> 
> @angelbarrera92 what do you think about the release? Minor or patch? Personally, I would go for a minor since we are upgrading an existing component.

Uhmmmm. If we include (directly or indirectly) a new feature (weave from 2.5 to 2.6 includes new features), increase a minor version, otherwise increase the patch number.

**Comment by angelbarrera92** [comment]
Should we close this PR as we are going to deprecate weave?

**Comment by lnovara** [comment]
I agree, let's close this PR.

**Pull request closed without merging**